python/ray/autoscaler/subprocess_output_util.py

```diff
@@ -234,10 +233,6 @@ def _run_and_process_output(cmd,
     # See implementation note #1
 
     if use_login_shells:
-        if stdout_file is None:
-            stdout_file = subprocess.DEVNULL
-        if stderr_file is None:
-            stderr_file = subprocess.DEVNULL
```

Comment:
this swallows...

Reply:
Makes sense, but I'm not super familiar with this code path

Reply:
I don't think we can just remove these since I am not confident that this default is never used, the proper way to resolve this is to just replace `subprocess.DEVNULL` with `sys.STDOUT` probably.

Reply:
shouldn't we do `stdout -> sys.STDOUT` and `stderr -> sys.STDERR`?